aloha-core/src/main/scala/com/eharmony/aloha/reflect/RefInfoOps.scala

```diff
@@ -159,6 +169,36 @@ object RefInfoOps extends RefInfoOps[RefInfo] with EitherHelpers {
   def typeParams[A](implicit a: RefInfo[A]): List[RefInfo[_]] = a.typeArguments
 
   /**
+    * Attempt to determine if `possibleIterable` is an `Iterable[A]`.
+    *
+    * Covers only simple one type variable instances in Predef.  This is kind of a stopgap solution because the
+    * subtype operator `<:<` doesn't really work well for Manifests.  This is an issue because currently, Aloha uses
+    * Manifests rather than `TypeTag`s for type checking.  Comments in ClassManifestDeprecatedApis says:
+    *
+    * "''this part is wrong for punting unless the rhs has no type arguments, but it's better
+    * than a blindfolded pinata swing.''"
+    *
+    * @param a RefInfo instance of element type
+    * @param possibleIterable a possible Iterable
+    * @tparam A element type
+    * @tparam Iter possible iterable type.
+    * @return true if `possibleIterable` is definitely a `Iterable[A]`.  May have false negatives.
+    */
+  def isIterable[A, Iter](implicit a: RefInfo[A], possibleIterable: RefInfo[Iter]): Boolean = {
+    List(a) == possibleIterable.typeArguments &&
+      (Array[Class[_]](
```

Comment:
Actually, some of the included types are not guaranteed to be immutable.  I am going to change this to only work on immutable `scala.collection.immutable.Iterable` types.  I think I can get an exhaustive list for that but I'll try to use reflection instead.  I'll just have to watch out for some gotchas.  I'll change this today.